.github/workflows/bench-devnet.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Bench Devnet"
+
+on:
+  # Remove this before merging, we only want devnet tests on master
```

Comment:
As discussed [here](https://github.com/FuelLabs/fuels-ts/pull/3166#discussion_r1763085556)

> [!NOTE]
> Remove this before merging so that we only run the devnet workflow on master builds

We will only run this workflow on release